apps/blog/src/app/(pages)/tag/[tag]/page.tsx

```
@@ -22,5 +22,9 @@ export const metadata: Metadata = createMetadata({
 
 export default async function Tags({ params }: DynamicRouteParams) {
   const posts = await trpcServerSide.post.getPostCards();
-  return <TagsPage posts={posts} tag={params.tag} />;
+  return (
+    <HydrateClient>
+      <TagsPage posts={posts} tag={params.tag} />;
+    </HydrateClient>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)